tests/e2e/pom/shortcuts-settings-model.ts

```diff
@@ -0,0 +1,23 @@
+import { type Locator, type Page } from "@playwright/test";
+
+export class DevShortcutsSettingsPage {
+    readonly page: Page;
+    readonly shortcutsVisibilityCheckbox: Locator;
+
+    constructor(page: Page) {
+        this.page = page;
+        this.shortcutsVisibilityCheckbox = page.getByLabel("shortcuts-visibility-checkbox");
```

Comment:
Shouldn't this be the following?
```suggestion
        this.shortcutsVisibilityCheckbox = page.getByLabel("Display Shortcuts");
```

Reply:
Oh, you're right. Funny, that the tests did work...